engine/src/stack.h

```diff
@@ -300,6 +300,9 @@ class MCStack : public MCObject, public MCMixinObjectHandle<MCStack>
     
     // MW-2014-09-30: [[ ScriptOnlyStack ]] If true, the stack is a script-only-stack.
     bool m_is_script_only : 1;
+    
+    // BWM-2017-08-16: [[ Bug 17810 ]] Line endings for imported script-only-stack.
+    MCStringLineEndingStyle m_line_encoding_style;
```

Comment:
You can use
```
MCStringLineEndingStyle m_line_ending_style : 2;
```
This won't break the bitfield, and is a little more economical on space.